src/repositories/repoBranchesRepository.ts

```diff
@@ -8,6 +8,15 @@ export class RepoBranchesRepository extends BaseRepository {
     super(config, logger, 'RepoBranchesRepository', db.collection(config.get('repoBranchesCollection')));
   }
 
+  async getBuildDependencies(repoName: string) {
```

Comment:
nit: Could you add a return type to this just to keep us on the right path and because you already have those gorgeous types ready :)